posthog/hogql_queries/experiments/experiment_query_runner.py

```diff
@@ -146,7 +146,7 @@ def _get_experiment_query(self) -> ast.SelectQuery:
         # Finds when each user was first exposed to each experiment variant
         exposure_query = ast.SelectQuery(
             select=[
-                ast.Field(chain=["distinct_id"]),
+                ast.Alias(alias="aggregation_target", expr=ast.Field(chain=["person_id"])),
```

Comment:
I'd prefer `entity_id` as the alias here. Later, we might add support for a different aggregation target than the identifier we use to join with. F.ex, we use "person_id" as the identifier to join data together , but we allow users to analyze on a group level. So it makes sense to separate those concepts I think.

Reply:
@andehen Not strongly opposed. Updated with 9f454ae028